RijsMuseum/app/src/main/java/com/example/rijsmuseum/MainActivity.kt

```diff
@@ -54,9 +54,15 @@ class MainActivity : AppCompatActivity(), RecyclerAdapter.OnItemClickListener {
 
     override fun onItemClick(cDetailsArtObject: Collections.ArtObject) {
         supportFragmentManager.beginTransaction().apply {
+            //Bundle and Intent
+            val bundle = Bundle()
+            bundle.putString("objectNumber", cDetailsArtObject.objectNumber)
+            val detailsFrag = DetailsFragment()
+            detailsFrag.arguments = bundle
```

Comment:
try to make this nicer and combine the 4 rows into one. To make it more kotlin-y